cmd/deployment.go

```diff
@@ -93,7 +106,19 @@ func deploymentCreate(cmd *cobra.Command, args []string) error {
 	// Silence Usage as we have now validated command input
 	cmd.SilenceUsage = true
 
-	return deployment.Create(args[0], ws)
+	var executorValue string
+	switch executor {
+	case "local":
+		executorValue = "LocalExecutor"
+	case "celery":
+		executorValue = "CeleryExecutor"
+	case "kubernetes", "k8s":
+		executorValue = "KubernetesExecutor"
+	default:
```

Comment:
@andriisoldatenko wondering if we should remove the `default` fallback here and not specify an executor if they don't specify one. This will let the API determine the default executor, so we won't have to change it in multiple places.

Reply:
nice catch!

Reply:
@schnie one note: i can't send without, only with `config: ""` or similar. Otherwise i need to create 2 GraphQL queries.

Reply:
Gotcha, if you send an empty string does it default to celery at the api layer?

Reply:
when i'm trying to send empty config, in default scenario I get response like:

```
mutation CreateDeployment(
		$label: String!
		$type: String = "airflow"
		$workspaceId: Uuid!
		$config: JSON!
	) {
		createDeployment(
			label: $label
			type: $type
			workspaceUuid: $workspaceId
            config: $config
		) {
			id
			type
			label
			releaseName
			version
      config
			createdAt
			updatedAt
		}
	}
```

**Response:**

```
{
  "data": {
    "createDeployment": {
      "id": "cjyea2qp6010f0b78sbjk4nsm",
      "type": "airflow",
      "label": "test124",
      "releaseName": "solar-axis-9034",
      "version": "0.10.0-alpha.4",
      "config": {},
      "createdAt": "2019-07-22T10:59:27.785Z",
      "updatedAt": "2019-07-22T10:59:27.785Z"
    }
  }
}
```

Reply:
or when i'm trying to send config like `"executor": ""`, i see error from API:
```
{
  "errors": [
    {
      "message": "Cannot read property 'components' of undefined",
      "locations": [
        {
          "line": 7,
          "column": 3
        }
      ],
      "path": [
        "createDeployment"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      },
      "level": "ERROR",
      "timestamp": "2019-07-22T11:02:19"
    }
  ],
  "data": {
    "createDeployment": null
  }
}
```

Reply:
Ok, it looks like an empty string won't work - it has to be `undefined` for it to default currently. https://github.com/astronomer/houston-api/blob/master/src/lib/deployments/config/index.js#L120
https://lodash.com/docs/4.17.14#get

Reply:
`"config": {}` works, i'm using it for emulate default value.